src/autobind.cr

```diff
@@ -3,25 +3,52 @@ require "./autobind/parser"
 require "./autobind/type"
 
 cflags = [] of String
-header = nil
+header = ""
 remove_enum_prefix = remove_enum_suffix = false
+lib_name = "LibC"
+mod_name = nil
+Help = <<-USAGE
+usage : c2cr [--help] [options] <header.h>
+
+Some available options are:
+    -I<directory>   Adds directory to search path for include files
+    -D<name>        Adds an implicit #define
+
+In addition, the CFLAGS environment variable will be used, so you may set it
+up before compilation when search directories, defines, and other options
+aren't fixed and can be dynamic.
+
+The following options control how enum constants are cleaned up. By default
+the value is false (no cleanup), whereas true will remove matching patterns,
+while a fixed value will remove just that:
+    --remove-enum-prefix[=true,false,<value>]
+    --remove-enum-suffix[=true,false,<value>]
+
+The resulting library name can be controlled with the --lib-name argument,
+and it can be wrapped in a parent module with --module-name.
+    --lib-name="LibC"         (the default)
+    --parent-module="Library"
+If no module is specified, the resulting code is not wrapped in a parent
+module. If the library name is not specified, it will be called "LibC".
+USAGE
+UsageExitCode = 64
```

Comment:
Constants are usually snake uppercase, like `HELP` and `USAGE_EXIT_CODE`
But, why having this code vs. only `abort`?

Reply:
It's just something I do. One time I looked to see if there was any sort of standardization for exit codes and I found [this header file](https://gist.githubusercontent.com/bojanrajkovic/831993/raw/79d07934534ba03d1b21c78917b9a8b699d8d6fe/sysexits.h) so I've been making my exit codes match up to that, so that they're meaningful.

If you don't like it I can take it out, I just like the idea of having a standardized meaningful exit code, even if 99% of the time it is 64 lol